packages/web-twig/src/Resources/components/Accordion/README.md

```diff
@@ -0,0 +1,198 @@
+      Accordion Content
+    </AccordionContent>
+  </AccordionItem>
+  <AccordionItem id="AccordionItemExample1">
+    <AccordionHeader
+      id="AccordionItemExample1Header"
+      for="AccordionItemExample1Content"
+      slot="<Pill color='selected'>3</Pill>"
+      isOpen
+    >
+      Accordion Header
+    </AccordionHeader>
+    <AccordionContent id="AccordionItemExample1Content" labelId="AccordionItemExample1Header" isOpen>
+      Accordion Content
+    </AccordionContent>
+  </AccordionItem>
+</Accordion>
+```
+
+## Usage with open only item one at a time
+
+```html
+<Accordion id="AccordionExample">
+  <AccordionItem id="AccordionItemExample0">
+    <AccordionHeader
+      id="AccordionItemExample0Header"
+      for="#AccordionItemExample0Content"
+      slot="<Link href='#'>Link</Link><Pill color='selected'>3</Pill>"
+    >
+      Accordion Header
+    </AccordionHeader>
+    <AccordionContent id="AccordionItemExample0Content" labelId="AccordionItemExample0Header" parent="AccordionExample">
+      Accordion Content
+    </AccordionContent>
+  </AccordionItem>
+  <AccordionItem id="AccordionItemExample1">
+    <AccordionHeader
+      id="AccordionItemExample1Header"
+      for="AccordionItemExample1Content"
+      slot="<Pill color='selected'>3</Pill>"
+    >
+      Accordion Header
+    </AccordionHeader>
+    <AccordionContent id="AccordionItemExample1Content" labelId="AccordionItemExample1Header" parent="AccordionExample">
+      Accordion Content
+    </AccordionContent>
+  </AccordionItem>
+</Accordion>
+```
+
+## Usage without lexer:
+
+```twig
+{% embed "@spirit/accordion.twig" with { props: {
+  id: 'AccordionExample',
+} } %}
+    {% block content %}
+        {% embed "@spirit/accordionItem.twig" with { props: {
+          id: `AccordionItemExample`,
+        } } %}
+            {% block content %}
+                {% embed "@spirit/accordionHeader.twig" with { props: {
+                  id: 'AccordionItemExampleHeader',
+                  for: 'AccordionItemExampleContent',
+                  slot: '<Link href="#">Link</Link><Pill color="selected">3</Pill>',
+                  isOpen: true,
+                } } %}
+                    {% block content %}
+                        Accordion Header
+                    {% endblock %}
+                {% endembed %}
+                {% embed "@spirit/accordionContent.twig" with { props: {
+                  id: 'AccordionItemExampleContent',
+                  labelId: 'AccordionItemExampleHeader',
+                  parent: 'AccordionExample',
+                  isOpen: true,
+                } } %}
+                    {% block content %}
+                        Accordion Content
+                    {% endblock %}
+                {% endembed %}
+            {% endblock %}
+        {% endembed %}
+    {% endblock %}
+{% endembed %}
+```
+
+## API
+
+The Accordion itself consists of many components which cannot be used independently.
```

Comment:
The number is four, so I'd say `several` of `a few` rather than `many`… 🙂

Reply:
ok, fixed 🙂